exp/capacity-planning.md

```diff
@@ -0,0 +1,96 @@
+When planning your Anbox Cloud deployment, you should start by estimating how much capacity you need to be able to provide your application to your users and how many users you expect. Based on this estimate, you can then size your deployment and figure out how many cluster nodes you need and what resources they should have.
+
+In your calculation, you should consider the following aspects:
+
+- Application resources - How much CPU, memory and disk space does your application need? Will the application use hardware- or software-based video encoding, and if it uses hardware-based encoding, how much GPU capacity is needed?
```

Comment:
No, `GPU capacity` is ok here as `GPU slots` is already too specific to what AMS does. `GPU capacity` later maps into `GPU slots` but at the high level we should not use it as it will only confuse people.

Reply:
That makes sense to me.